src/main/java/com/google/sps/data/DataUtil.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+import com.google.gson.Gson;
+
+public class DataUtil {
+    public static final Gson PARSER = new Gson();
+    public static enum video_state {
```

Comment:
VideoState

Also, I don't think this should be included in the DataUtil file unless it is only used inside this file.

Reply:
We decided to get rid of the DataUtil file as per Tianxiao's request